components/reaction-badge.json

```diff
@@ -0,0 +1,34 @@
+{
+  "reaction-badge": {
+    "comment": "Applied to reaction buttons",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=5585%3A13103",
+    "primary": {
+      "#normal": {
+        "background": "@theme-background-secondary-normal",
+        "text": "@theme-text-primary-normal",
+        "border": "@theme-outline-secondary-normal"
+      },
+      "#hovered": {
+        "background": "@theme-background-secondary-hover",
+        "text": "@theme-text-primary-normal",
+        "border": "@theme-outline-secondary-normal"
+      },
+      "#pressed": {
+        "background": "@theme-background-secondary-active",
+        "text": "@theme-text-primary-normal",
+        "border": "@theme-outline-secondary-normal"
+      },
+      "reacted": {
```

Comment:
All the children of an element should either be UI states (#normal/#hovered/#pressed) or objects, we can't support both (it will cause trouble for Qt clients). I suggest maybe change to:
"#normal": {
        "background": "@theme-background-secondary-normal",
        "unreacted-text": "@theme-text-primary-normal",
        "reacted-text": "@theme-text-accent-normal",
        "border": "@theme-outline-secondary-normal"
      },